x-pack/plugins/global_search/public/index.ts

```diff
@@ -31,4 +31,4 @@ export {
   GlobalSearchPluginStart,
   GlobalSearchResultProvider,
 } from './types';
-export { GlobalSearchFindOptions } from './services/types';
+export { SearchServiceStart, GlobalSearchFindOptions } from './services';
```

Comment:
I don't see any usage of the added `SearchServiceStart` export, and I think the plugin contracts are enough. I think this can be removed, or did I miss something?